### ISSUE TITLE
Words are not wrapped correctly in add-on details page - statistics section 

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#4253]
Prerequisites: Locale set to German. Italian, Spanish

STR:

1. Go to AMO homepage in responsive design mode (~750 - 1000 width) 
2. Select an Add-on to view its listing page (i.e. https://addons-dev.allizom.org/de/firefox/addon/awesome-screenshot-plus-/)
3. Observe the statistics section in the right corner

Expected result:
There are no layout issues

 Actual result:
 Text is not wrapped correctly

Notes:
- reproduces on AMO dev with FF58, Win10 and Android 7.0

![image](https://user-images.githubusercontent.com/31961530/35521663-fa70b9a6-0522-11e8-9bd5-12df3f4e66c2.png)

**Comment by tofumatt** [comment]
The text is actually wrapped correctly but there is no indication that it's wrapping–a hyphen would be good to indicate the text doesn't fit on one line.

Otherwise it's up to the localisers to find a more concise wording, if one exists. The mocks for this don't leave us much room.

**Comment by tsl143** [comment]
@tofumatt, we can use `hyphens` CSS property for this :)

**Comment by tofumatt** [comment]
This was fixed by #4495.

**Comment by AlexandraMoga** [comment]
Verified as fixed on AMO stage with FF58, Win10x64

![image](https://user-images.githubusercontent.com/31961530/37086831-3d07a522-2201-11e8-9c23-ba0f8404a551.png)
